parts/common/components.json

```diff
@@ -1117,23 +1117,56 @@
       }
     },
     {
-      "name": "cni-plugins",
+      "name": "containernetworking-plugins",
```

Comment:
@Devinwong  any potential issue with a name change ?

Reply:
[replying after 8 months 😄]
Should be fine. Just need to change the names in the scripts too.
For example:
```
installCNI() {
    # Old versions of VHDs will not have components.json. If it does not exist, we will fall back to the hardcoded download for CNI.
    # Network Isolated Cluster / Bring Your Own ACR will not work with a vhd that requres a hardcoded CNI download.
    if [ ! -f "$COMPONENTS_FILEPATH" ] || ! jq '.Packages[] | select(.name == "cni-plugins")' < $COMPONENTS_FILEPATH > /dev/null; then
        echo "WARNING: no cni-plugins components present falling back to hard coded download of 1.4.1. This should error eventually"
```